build.gradle

```diff
@@ -1,35 +1,31 @@
 import java.nio.file.*;
 
 ext.fireflyPath = "$rootDir.path/../firefly/"
-
-
+ext.onlinehelp_dir = "$rootDir/../suit-onlinehelp/"
 
 allprojects {
-  defaultTasks 'build'
-  apply from: "$fireflyPath/buildScript/init.gincl"
-  apply from: "$rootDir.path/buildScript/init.gincl"
+  defaultTasks 'preselect'
 }
 
 subprojects {
-
   apply plugin: "java"
-  apply from: "$fireflyPath/buildScript/depends.gincl"
-  apply from: "$rootDir.path/buildScript/depends.gincl"
-
+  apply from: "$fireflyPath/buildScript/global.gincl"
+  apply from: "$fireflyPath/buildScript/tasks.gincl"
+  apply from: "$rootDir.path/buildScript/global.gincl"
 }
 
-
-
-apply from: "$fireflyPath/buildScript/utils.gincl"
-
 task purge << {
   println('Removes all build directories.')
```

Comment:
It attracted my attention that "purge" task is missing description in suit, while the description is present in firefly.
Using `println`, but no description. Same with clean task below.

I found tasks help with descriptions very helpful. These are a few tasks with no description in suit:
onlinehelp
clean
preselect
purge
testJar
webapp